products/binaryauthorization/api.yaml

```diff
@@ -142,10 +142,10 @@ objects:
       - !ruby/object:Api::Type::NameValues
         name: clusterAdmissionRules
         description: |
-          Admission policy whitelisting. A matching admission request will
-          always be permitted. This feature is typically used to exclude Google
-          or third-party infrastructure images from Binary Authorization
-          policies.
+          Per-cluster admission rules. There can be at most one admission rule
```

Comment:
Oh, thanks. Want to also add some information on what an admission rule actually is? https://cloud.google.com/binary-authorization/docs/reference/rest/v1beta1/projects.policy#admissionrule

Reply:
Done. Do you mind reviewing the new text? I *think* I captured it correctly, I wasn't happy with the phrasing in the whitelist policy so I redid that.